package.json

```diff
@@ -34,7 +34,9 @@
   },
   "devDependencies": {
     "browserify": "^13.0.0",
+    "choo": "^1.1.0",
     "standard": "^6.0.8",
-    "tape": "^4.5.1"
+    "tape": "^4.5.1",
+    "yo-yo": "^1.2.0"
```

Comment:
should probably also add a regression test for `yo-yo`; perhaps in a separate patch